.pre-commit-config.yaml

```
@@ -28,10 +28,20 @@ repos:
       - id: requirements-txt-fixer
       - id: mixed-line-ending
         args: ["--fix=auto"]
-  - repo: https://github.com/google/yapf
-    rev: v0.43.0
+  - repo: https://github.com/psf/black
+    rev: 23.11.0
     hooks:
-      - id: yapf
+    - id: black
+      language_version: python3
+  - repo: https://github.com/pycqa/isort
```

I'm not sure if this is the exact correct syntax for this, but Black defaults to a line length of 88. [PEP 8 subscribes a max line length as 79](https://peps.python.org/pep-0008/#maximum-line-length), so it should be set in Black as well.

```suggestion
    - id: black
      language_version: python3
      args: ["--line-length=79"]
  - repo: https://github.com/pycqa/isort
```